### ISSUE TITLE
Quantum extractor failing - lua error

### ISSUE THREAD

**Issue by damianb** [sayterdarkwynd/FrackinUniverse#681]
```
[12:02:54.533] [Error] Exception while creating lua context for scripts '('/objects/power/isn_sharedpowerscripts.lua', '/objects/isn_sharedobjectscripts.lua', '/objects/generic/extractionlab_common.lua')': (LuaException) Error code 3, [string "/objects/generic/extractionlab_common.lua"]:1037: '}' expected (to close '{' at line 767) near 'end'
[0] 7ff759ae4403 Star::captureStack
[1] 7ff759ae318e Star::StarException::StarException
[2] 7ff759ab7a91 Star::LuaEngine::handleError
[3] 7ff759ab4292 Star::LuaEngine::compile
[4] 7ff759ab420a Star::LuaEngine::compile
[5] 7ff75a0e2a6f Star::LuaRoot::ScriptCache::loadScript
[6] 7ff75a0e29a5 Star::LuaRoot::ScriptCache::loadContextScript
[7] 7ff75a0e26ee Star::LuaRoot::createContext
[8] 7ff75a0d0cf3 Star::LuaBaseComponent::init
[9] 7ff759d5b4be Star::Object::init
[10] 7ff75a0b6e7c Star::ContainerObject::init
[11] 7ff759fd188a Star::WorldGenerator::initEntity
[12] 7ff75a04e94c Star::WorldStorage::loadSectorToLevel
[13] 7ff75a04e1a3 Star::WorldStorage::loadSector
[14] 7ff759f95ce7 <lambda_cb1d062c3d3843942fecf4e958af384b>::operator()
[15] 7ff759f95fb4 <lambda_cb1d062c3d3843942fecf4e958af384b>::operator()
[16] 7ff759f95fb4 <lambda_cb1d062c3d3843942fecf4e958af384b>::operator()
[17] 7ff759f95ec4 <lambda_cb1d062c3d3843942fecf4e958af384b>::operator()
[18] 7ff759f95fb4 <lambda_cb1d062c3d3843942fecf4e958af384b>::operator()
[19] 7ff759f95ec4 <lambda_cb1d062c3d3843942fecf4e958af384b>::operator()
[20] 7ff759f95fb4 <lambda_cb1d062c3d3843942fecf4e958af384b>::operator()
[21] 7ff759f9757d Star::WireProcessor::loadNetwork
[22] 7ff759f978dd Star::WireProcessor::process
[23] 7ff75a03de8a Star::WorldServer::update
[24] 7ff75a045c42 Star::WorldServerThread::update
[25] 7ff75a0456a8 Star::WorldServerThread::run
[26] 7ff759ae0f2e Star::ThreadImpl::runThread
[27] 7ffb4c648364 BaseThreadInitThunk
[28] 7ffb4f0070d1 RtlUserThreadStart
```

Appears to be due to the merge of "MoreFarming" and "Yes Chef!"

Looks like all the recipes have some serious { } mismatch issues.

**Comment by damianb** [comment]
Opened PR #681 to fix this.

**Comment by damianb** [comment]
Might also be fixed by #683.
